### DISCUSSION TITLE
Ability to Add multiple alerts for a panel

### DISCUSSION THREAD

**Discussion by achinthagunasekara** [grafana/grafana#54504] [Alerting]
Currently we can only add one alert per panel.

![Screen Shot 2022-08-31 at 9 32 10 am](https://user-images.githubusercontent.com/3791781/187561484-de7125da-bacd-4d2f-9914-7c182099f6f8.png)

It would be nice to be able to send a message after `x` amount of time and then may be page after `x + y` amount of time.  Right now only way to do this is to create a duplicate panel.

Can we please add the ability to add multiple alerts per panel with different thresholds and different `Send To` methods

thanks,

**Comment by josephthanaraja** [comment]
Yes I hope that is possible and we have done the same sometime back. Please refer the below.

you can create multiple queries in that single panel and use the "multiple conditions" feature in the alert section. They will be under one alert.

However, you can use the [grafana notification templating] in order to distinguish your alert conditions as seen in the pic below.
<img width="651" alt="Screen Shot 2022-09-01 at 9 22 11 AM" src="https://user-images.githubusercontent.com/52676992/187925019-660fa75d-b491-4e67-a263-2b344399bf0e.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
